features/fork.feature

```diff
@@ -167,10 +170,10 @@ Feature: hub fork
       }
       """
     When I run `hub fork`
-    Then the exit status should be 128
-    And the stderr should contain exactly:
+    Then the exit status should be 0
+    And the stdout should contain exactly:
       """
-      fatal: remote mislav already exists.\n
+      existing remote: mislav\n
```

Comment:
Why has the error message in this case changed?

Reply:
because it's now coming from hub, not git. This unifies `Unrelated remote already exists` with other related scenarios like `Related fork and related remote already exist`, `Related fork and related remote, but with differing protocol, already exist`, ...